pkg/version/version.go

```diff
@@ -13,19 +13,23 @@ var (
 	buildStatus      = "unknown"
 	buildTag         = "unknown"
 
+	// Minimum supported mesh version (nil (all), "v1_0", "v1_1", "v2_0", "v2_1" etc)
+	minimumSupportedVersion = ""
```

Comment:
empty means all :)

Reply:
ah yes, not nil.  That comment is also like that in the 2.0 version